ResidentsSocialCarePlatformApi.Tests/V1/Controllers/VisitInformationControllerTests.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using ResidentsSocialCarePlatformApi.V1.Boundary.Responses;
+using ResidentsSocialCarePlatformApi.V1.Controllers;
+using ResidentsSocialCarePlatformApi.V1.UseCase.Interfaces;
+
+namespace ResidentsSocialCarePlatformApi.Tests.V1.Controllers
+{
+    public class VisitInformationControllerTests
+    {
+        private VisitInformationController _classUnderTest;
+        private Mock<IGetVisitInformationByPersonId> _mockGetCaseVisitInformationByPersonIdUseCase;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockGetCaseVisitInformationByPersonIdUseCase = new Mock<IGetVisitInformationByPersonId>();
+
+            _classUnderTest = new VisitInformationController(_mockGetCaseVisitInformationByPersonIdUseCase.Object);
+        }
+
+        [Test]
+        public void GetCaseNote_WhenThereIsAMatchingCaseNoteId_ReturnsCaseNoteInformation()
+        {
+            var formattedDateTime = new DateTime(2020, 4, 1, 20, 30, 00).ToString("s");
```

Comment:
```suggestion
            readonly string formattedDateTime = new DateTime(2020, 4, 1, 20, 30, 00).ToString("s");
```

Reply:
It seems to compile fine but both vscode and rider are flagging it.

Rider: `Local function must declare a body because it is not marked 'static extern'.`

VsCode: `The modifier 'readonly' is not valid for this item`.

I'm still learning lots about C#, seems like only fields and not variables can be readonly [Docs here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/readonly)

Reply:
My bad! Apologies - been a while since I c#'d :)